demo/discourse.py

```diff
@@ -9,9 +9,11 @@ def __init__(self, url: Text):
         self.url = url
 
     @staticmethod
-    def get_discourse_links(topics: List[Dict[Text, Any]], index: int):
-        doc_url = f"https://forum.rasa.com/t/{topics[index]['slug']}/{str(topics[index]['id'])}"
-        forum = f"- [{topics[index]['title']}]({doc_url})"
+    def get_discourse_links(topics: Optional[List[Dict[Text, Any]]], index: int):
+        forum = None
+        if topics:
+            doc_url = f"https://forum.rasa.com/t/{topics[index].get('slug')}/{str(topics[index].get('id'))}"
+            forum = f"- [{topics[index]['title']}]({doc_url})"
```

Comment:
```suggestion
            forum = f"- [{topics[index].get('title')}]({doc_url})"
```
would you mind doing a search for any more of these?

Reply:
Yep!  I will do a search on these type of cases.